test/clustertest/main.cpp

```diff
@@ -72,7 +73,9 @@ int main(int argc, char* argv[]) {
     if (args.isSet("-threads")) {
         threads = SToInt(args["-threads"]);
     }
-
+    if (args.isSet("-dieOnFailure")) {
+            dieOnFailure = true;
```

Comment:
I think we're using double indentation here instead of just one:

```suggestion
        dieOnFailure = true;
```